membership/build.gradle

```diff
@@ -1,18 +1,17 @@
 plugins {
     id 'corda-api.common-library'
     id 'corda.common-publishing'
-    id 'org.jetbrains.kotlin.jvm'
```

Comment:
Seeing as you've managed to remove Kotlin _completely_, can you also add:
```
id 'corda.java-only'`
```
here please? This will prevent people adding new Kotlin classes back in.